mne/stats/regression.py

```diff
@@ -134,3 +137,193 @@ def _fit_lm(data, design_matrix, names):
         mlog10_p_val[predictor] = -np.log10(p_val[predictor])
 
     return beta, stderr, t_val, p_val, mlog10_p_val
+
+
+def linear_regression_raw(raw, events, event_id=None, tmin=-.1, tmax=1,
+                          covariates=None, reject=None, tstep=1.,
+                          decim=1, picks=None, solver='pinv'):
+    """Estimate regression-based evoked potentials/fields by linear modelling
+    of the full M/EEG time course, including correction for overlapping
+    potentials and allowing for continuous/scalar predictors. Internally, this
+    constructs a predictor matrix X of size n_samples * (n_conds * window
+    length), solving the linear system Y = bX and returning b as evoked-like
+    time series split by condition.
+
+    See Smith, N. J., & Kutas, M. (2015). Regression-based estimation of ERP
```

Comment:
see how these should be done here: https://github.com/numpy/numpy/blob/master/doc/HOWTO_DOCUMENT.rst.txt